<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>901</width>
      <height>521</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>881</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter panier</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>231</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
font: 10pt &quot;MS Shell Dlg 2&quot;;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Reference Panier :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>310</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
font: 10pt &quot;MS Shell Dlg 2&quot;;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Prix Panier  :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>190</y>
          <width>221</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
font: 10pt &quot;MS Shell Dlg 2&quot;;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>CIN Client :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_ref">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>100</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_cin">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>210</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prix">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>320</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>140</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;NSimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier_2">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>280</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;NSimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>10</y>
          <width>341</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
font: 24pt &quot;Sans Serif Collection&quot;;
border-color: rgb(255, 170, 0);
</string>
        </property>
        <property name="text">
         <string>Ajouter un Panier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QTableView" name="tab_panier">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>581</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_recherche">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>30</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(216, 191, 0);</string>
        </property>
        <property name="text">
         <string>Recherche</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_recherche">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>100</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
background-color: rgb(255, 255, 207);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter_2">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>160</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(214, 196, 0);
</string>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_recherche_2">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>230</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Black</family>
          <pointsize>20</pointsize>
          <weight>10</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 200, 0);
color: rgb(255, 83, 83);
font: 87 20pt &quot;Segoe UI Black&quot;;</string>
        </property>
        <property name="text">
         <string>Tri :</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_ref">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>300</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>14</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Ref</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_cin">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>340</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>14</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_price">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>370</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>14</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_stats">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>350</y>
          <width>201</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Script</family>
          <pointsize>36</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(216, 191, 0);
font: 75 36pt &quot;Script&quot;;</string>
        </property>
        <property name="text">
         <string>Stats</string>
        </property>
       </widget>
       <widget class="QPushButton" name="qr">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>380</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>QR CODE</string>
        </property>
       </widget>
       <widget class="QLabel" name="QRCODE_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>340</y>
          <width>111</width>
          <height>111</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>90</y>
          <width>361</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
font: 10pt &quot;MS Shell Dlg 2&quot;;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Donner la ref à supprimer :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>250</y>
          <width>221</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;NSimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_supp_ref">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>110</y>
          <width>201</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
